test/functional/aws_standalone_vpc.go

```diff
@@ -0,0 +1,573 @@
+	}
+
+	if testErrors.isValid() {
+		t.Fatal(testErrors.Error())
+	}
+}
+
+// verify that the standalone vpc is created
+func verifyVpc(session *ec2.EC2, clusterTag, expectedCidr string) (*ec2.Vpc, error) {
+	newErr := &networkConfigTestError{
+		vpcError: []string{},
+	}
+
+	// filter vpcs by name and integreatly tag
+	describeVpcs, err := session.DescribeVpcs(&ec2.DescribeVpcsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("tag:Name"),
+				Values: []*string{aws.String(vpcTagName)},
+			},
+			{
+				Name:   aws.String("tag:integreatly.org/clusterID"),
+				Values: []*string{aws.String(clusterTag)},
+			},
+		},
+	})
+	if err != nil {
+		newErr.vpcError = append(newErr.vpcError, fmt.Sprintf("could not find vpc: %v", err))
+		return nil, newErr
+	}
+
+	// only one vpc is expected
+	vpcs := describeVpcs.Vpcs
+	if len(vpcs) != 1 {
+		newErr.vpcError = append(newErr.vpcError, fmt.Sprintf("expected 1 vpc but found %d", len(vpcs)))
+		return nil, newErr
+	}
+
+	// cidr blocks should match
+	vpc := vpcs[0]
+	foundCidr := aws.StringValue(vpc.CidrBlock)
+	if foundCidr != expectedCidr {
+		newErr.vpcError = append(newErr.vpcError, fmt.Sprintf("cidr blocks not equal, expected %s but got %s", expectedCidr, foundCidr))
+	}
+
+	return vpc, newErr
+}
+
+// verify that the vpc subnets are created
+func verifySubnets(session *ec2.EC2, clusterTag, expectedCidr string) ([]*ec2.Subnet, error) {
+	newErr := &networkConfigTestError{
+		subnetsError: []string{},
+	}
+
+	// filter subnets by name and integreatly tag
+	describeSubnets, err := session.DescribeSubnets(&ec2.DescribeSubnetsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("tag:Name"),
+				Values: []*string{aws.String(subnetTagName)},
+			},
+			{
+				Name:   aws.String("tag:integreatly.org/clusterID"),
+				Values: []*string{aws.String(clusterTag)},
+			},
+		},
+	})
+	if err != nil {
+		newErr.subnetsError = append(newErr.subnetsError, fmt.Sprintf("could not describe subnets: %v", err))
+		return nil, newErr
+	}
+
+	// parse the vpc cidr block from the createStrategy for _network
+	subnets := describeSubnets.Subnets
+	_, cidr, err := net.ParseCIDR(expectedCidr)
+	if err != nil {
+		newErr.subnetsError = append(newErr.subnetsError, fmt.Sprintf("could not parse vpc cidr block: %v", err))
+		return subnets, newErr
+	}
+	cidrMask, _ := cidr.Mask.Size()
+
+	// verify the subnet masks for the subnets are one bit bigger
+	// than the vpc subnet mask
+	for _, subnet := range subnets {
+		_, subnetCidr, err := net.ParseCIDR(aws.StringValue(subnet.CidrBlock))
+		if err != nil {
+			newErr.subnetsError = append(newErr.subnetsError, fmt.Sprintf("could not parse subnet mask for vpc subnets: %v", err))
+			return subnets, newErr
+		}
+		subnetCidrMask, _ := subnetCidr.Mask.Size()
+		if subnetCidrMask != cidrMask+1 {
+			newErr.subnetsError = append(newErr.subnetsError, fmt.Sprintf("unexpected subnet mask size for vpc subnets: %v", err))
```

Comment:
I think we should call out what we expected to find here, something like
```suggestion
			newErr.subnetsError = append(newErr.subnetsError, fmt.Sprintf("subnet mask expect to be 1 bit greater than vpc subnet mask, found : %d, expected %d, subnetCidrMask, cidrMask + 1))
```